source/backend/dal/Repositories/Interfaces/IPropertyActivityRepository.cs

```diff
@@ -12,10 +12,14 @@ public interface IPropertyActivityRepository : IRepository<PimsPropertyActivity>
 
         PimsPropertyActivity GetActivity(long activityId);
 
+        IList<PimsPropertyActivity> GetActivitiesByManagementFile(long managementFileId);
+
         PimsPropertyActivity Create(PimsPropertyActivity propertyActivity);
 
         PimsPropertyActivity Update(PimsPropertyActivity propertyActivity);
 
         bool TryDelete(long activityId);
+
+        bool TryDeleteFile(long activityId, long managementFileId);
```

Comment:
I believe we were going to call this "TryDeleteFileActivity" to avoid confusion on whether this method deletes the whole file (which it shouldn't)